.github/ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,25 @@
 <!--
-- Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or
-  https://gitter.im/strongloop/loopback
+- Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs
+  or https://gitter.im/strongloop/loopback
 
-- Immediate support is available through our subscription plans, see
-  https://strongloop.com/api-connect-faqs/
+- Immediate support is available through our subscription plans,
+  see https://strongloop.com/api-connect-faqs/
 -->
 
-### Bug or feature request
-
-<!--
-Mark your choice with an "x" (eg. [x], NOT [*]).
--->
-
-- [ ] Bug
-- [ ] Feature request
-
-### Description of feature (or steps to reproduce if bug)
-
-
-
-### Link to sample repo to reproduce issue (if bug)
-
+### Bug/Feature request
 
+<!-- Description of feature
+or steps to reproduce/link to sample repo if bug -->
 
 ### Expected result
 
+<!-- Include actual results if bug -->
 
+### Additional information
 
-### Actual result (if bug)
-
-
-
-### Additional information (Node.js version, LoopBack version, etc)
-
+```shell
+node -e 'console.log(process.platform, process.arch, process.versions.node)'
+```
 
+- Above command output: <!-- Paste output of command shown above here -->
+- LoopBack version: <!-- 2.x/3.x -->
```

Comment:
Should we ask users to run `npm ls --prod --depth 0 | grep loopback` to show us exact versions of all loopback-related modules they are using?